samples/CGCatalog/CGCatalog.vsimporter/CGCatalog-WinStore10/CGCatalog-Debug-xcvars.txt

```diff
@@ -0,0 +1,110 @@
+ALWAYS_SEARCH_USER_PATHS = NO
```

Comment:
please delete these two files with a commit.  The change looks good to me otherwise.

---
In reply to: [67958793](https://github.com/Microsoft/WinObjC/pull/566#discussion_r67958793) [](ancestors = 67958793)